.circleci/config.yml

```diff
@@ -103,11 +100,11 @@ jobs:
                 name: Run integration tests
                 command: |
                     . venv/bin/activate
-                    pip install --upgrade pip
-                    pip install -r tests/requirements.txt
```

Comment:
This step is run after the step above that already installs everything, 🔪

Reply:
Good catch :)